h2o-core/src/main/java/water/HeartBeatThread.java

```diff
@@ -102,6 +102,7 @@ public void run() {
       // persistent KV pairs are stored
       hb.set_free_disk(H2O.getPM().getIce().getUsableSpace());
       hb.set_max_disk (H2O.getPM().getIce().getTotalSpace() );
+      hb.set_dkv(Cleaner.Histo.current(false).dkv());
```

Comment:
What is the purpose of populating a transient field in the heartbeat structure? Heartbeats are meant to be exchanged between the node. This seems like like a misuse to me.

Reply:
it's hack, was used on single node only. No plan to merge this.